bundles/org.openhab.io.transport.modbus/src/main/java/org/openhab/io/transport/modbus/ModbusBitUtilities.java

```diff
@@ -205,10 +220,10 @@ public static DecimalType extractStateFromRegisters(ModbusRegisterArray register
      * NUL byte encountered.
      *
      * @param registers list of registers, each register represent 16bit of data
-     * @param index zero based register index. Registers are handled as 16bit registers,
-     *            this parameter defines the starting register.
-     * @param length maximum length of string in 8bit characters.
-     * @param charset the character set used to construct the string.
+     * @param index     zero based register index. Registers are handled as 16bit registers,
```

Comment:
Please apply the ESH formatter profile again in your eclipse installation and reformat this file

Reply:
I have confirmed that the formatter is active but still get the same result. Where would I find up-to-date formatter definition? 

Is the only option to reset the workspace? :(

Reply:
just chose another one and then ESH again and apply. It's just a bugged Eclipse

Reply:
Oh! Did not know that...

Whole modbus code base formatted in 39ff5a4.